gazelle/python/BUILD.bazel

```diff
@@ -16,9 +15,9 @@ go_library(
         "std_modules.go",
         "target.go",
     ],
-    data = [
-        ":parse",
-        ":std_modules",
+    embedsrcs = [
+        "parse.py",
+        "std_modules.py",
```

Comment:
To me this change would be fine if we would have previously had
```
data = [
    ":parse.py",
    ":std_modules.py",
]
```

The `py_binary` usage in here makes use of the default Python toolchain that is registered in the root module/workspace.

Hence the change made here is not equivalent. It would be if we had a `current_py_toolchain` in runtime gazelle deps and if we located that via the runfiles API.